cmd/launcher-pummel/README.md

```diff
@@ -0,0 +1,21 @@
+# Osquery Server Load Testing
+
+## Building the tool
+
+From the root of the repository, run the following:
+
+```
+make deps
+make launcher-pummel
+./build/launcher-pummel --help
+```
+
+## Usage Instructions
+
+```
+./build/launcher-pummel \
+  --host_path=./path/to/my/host/templates \
+	--server_url=fleet.acme.co \
+	--enroll_secret=mB3XE5wwLt3YryD9FAanjwhm02HoOqll \
```

Comment:
would love to see enroll secret path instead.

Reply:
Does the path allow this to be more easily used for cloud/fleet testing?

Reply:
Yes, because we generally save the secret to a file, and our configurations all use the `enroll_secret_path` field. 

At least that's been my personal expectation so far.

Reply:
if we wanted to make it more like launcher (which has both with an order of precedence), we could accept both: https://github.com/kolide/launcher/blob/bf16180aa067695b7cd6befa5a2a198a8dbf1696/cmd/launcher/launcher.go#L123-L132